src/three/GLTFExtensionLoader.js

```diff
@@ -0,0 +1,43 @@
+import { DefaultLoadingManager } from 'three';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+import { LoaderBase } from '../base/LoaderBase.js';
+
+export class GLTFExtensionLoader extends LoaderBase {
+
+	constructor( manager = DefaultLoadingManager ) {
+
+		super();
+		this.manager = manager;
+
+	}
+
+	parse( buffer ) {
+
+		return new Promise( ( resolve, reject ) => {
+
+			const manager = this.manager;
+			let loader = manager.getHandler( 'path.gltf' ) || manager.getHandler( 'path.glb' );
+
+			if ( ! loader ) {
+
+				loader = new GLTFLoader( manager );
+				loader.crossOrigin = this.crossOrigin;
+				loader.withCredentials = this.withCredentials;
+
+			}
```

Comment:
This still doesn't look right -- the LoaderBase doesn't include any `crossOrigin` or `withCredentials` fields. I think this should follow the same pattern that [B3DMLoader uses](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/5b0d133d28b62cefe2bb30a280303042f554c2bf/src/three/B3DMLoader.js#L22-L40) for adjusting the GLTFLoader settings from `this.fetchOptions`. This looks like the last change!